openapi/paths/aml-checks@{id}.yaml

```
@@ -0,0 +1,29 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - AML
+  summary: Retrieve results of a Customer's AML check
```

nitpick: good to keep these short

also, no need to capitalize customer

```suggestion
  summary: Retrieve a customer's AML check
```